### PR TITLE
Fetch the support address from the API, for #63

### PR DESCRIPTION
**Pull request by joseconsador**
This addresses the issue raised in #63 .
Calls the API to get the default support address and uses that, instead of always using support@domain .

/cc @jwswj @miogalang @mmolina 
### References
- Jira link: https://zendesk.atlassian.net/browse/MI-59
- Github Issue: #63 
### Risks
- Medium - tickets will not go through the contact form if a support email is not retrieved and set

### CODE REVIEW

File: src/app/code/community/Zendesk/Zendesk/Model/Api/SupportAddresses.php [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        foreach ($this->all() as $recipient_address) {
+            if ($recipient_address['default']) {
+                $address = $recipient_address;
+                break;
+            }
+        }
+
+        return $address;
+    }
+}
```
Comment (miogalang) [inline src/app/code/community/Zendesk/Zendesk/Model/Api/SupportAddresses.php]: Pls add newline at EOF

Comment (joseconsador) [inline src/app/code/community/Zendesk/Zendesk/Model/Api/SupportAddresses.php]: :haircut:

**Comment by miogalang** [comment]
:+1:

File: src/app/code/community/Zendesk/Zendesk/Model/Api/SupportAddresses.php [hunk 2]
```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * Copyright 2012 Zendesk.
```
Comment (jwswj) [inline src/app/code/community/Zendesk/Zendesk/Model/Api/SupportAddresses.php]: We don't need this in the headers of all the files. It just gets out of date...

Comment (joseconsador) [inline src/app/code/community/Zendesk/Zendesk/Model/Api/SupportAddresses.php]: :haircut:

**Comment by jwswj** [comment]
:+1:

**Pull request merged**